app/coffeescripts/views/assignments/VeriCiteSettingsDialog.coffee

```diff
@@ -0,0 +1,68 @@
+define [
```

Comment:
Require both templates from the turnitin settings dialog (renamed generically) and render the right form.  This file is only 2 lines different from the turnitin one, and one of those lines is the class name.